src/autowiring/test/OnceTest.cpp

```diff
@@ -55,6 +55,8 @@ TEST(OnceTest, MultiLambdaPending) {
     while(go) {
       o += [&] { x++; };
       y++;
+      if (y % 10000 == 0)
+        std::this_thread::sleep_for(std::chrono::milliseconds(1));
```

Comment:
That sounds like a good idea to me.  It's unlikely that this sleep will affect that much but I try to avoid using them where possible as a matter of policy.

What would you like to do